bigtable/client/internal/instance_admin.cc

```diff
@@ -57,6 +57,26 @@ std::vector<btproto::Instance> InstanceAdmin::ListInstances(
   return result;
 }
 
+btproto::Instance InstanceAdmin::GetInstance(std::string const& instance_id,
+                                             grpc::Status& status) {
+  // Copy the policies in effect for the operation.
+  auto rpc_policy = rpc_retry_policy_->clone();
+  auto backoff_policy = rpc_backoff_policy_->clone();
+
+  // Build the RPC request, try to minimize copying.
+  btproto::GetInstanceRequest request;
+  // Set Instance ID
+  request.set_name(project_name_ + "/instances/" + instance_id);
+
+  // Call RPC call to get response
+  auto response = ClientUtils::MakeCall(
```

Comment:
You can simplify this even further, just `return ClientUtils::MakeCall(`, right?

Reply:
yes correct